ofcourse-templates/home.html

```diff
@@ -1,10 +1,9 @@
-<%inherit file="master.mak" />
-
-<div class="jumbotron">
-    <h1>${course['name']}</h1>
-    <p>${course['desc']}</p>
-</div>
-
+{% extends "master.html" %}
+{% block jumbotron %}
+  <h1>{{ course.name }}</h1>
+  <p>{{ course.desc }}</p>
+{% endblock %}
+{% block body %}
```

Comment:
You HAD to comment on a random line, instead of the offending lines...now GitHub won't hide it.